src/main/java/the_warlord/powers/FocusPunchPower.java

```diff
@@ -35,7 +30,7 @@ public int onAttacked(DamageInfo info, int damageAmount) {
         if (!damageTaken && info.owner != null && info.type != DamageInfo.DamageType.THORNS && info.type != DamageInfo.DamageType.HP_LOSS && info.owner != this.owner && damageAmount > 0) {
             flash();
             damageTaken = true;
-            addToBot(new ApplyPowerAction(owner, owner, new TensionPower(owner, 5)));
+            addToBot(new ApplyPowerAction(owner, owner, new TensionPower(owner, 2)));
```

Comment:
I'm wondering if we should store the amount of Tension you get in amount2. That way, multiple copies of this would stack. Currenly, no matter how many focus punches you play, you'll only ever get 2 Tension.